cube_facility_msgs/srv/StationaryVehicleRequest.srv

```diff
@@ -0,0 +1,11 @@
+cube_facility_msgs/StationaryVehicleWarningType stationary_vehicle_type
+cube_facility_msgs/RoadType road_type
+cube_facility_msgs/LanePosition lane_position 
+cube_facility_msgs/InformationQuality information_quality    
+bool ignition_terminal_15                                       # set to True for ignition on; False for ignition off
```

Comment:
`ignition_terminal_15` is an odd name IMHO. I would prefer `ignition` or `running`.

Reply:
alright